### PR TITLE
Fix crash

### PR DESCRIPTION
**Pull request by fioriNTU**
Fixing crash on the rendering

### CODE REVIEW

File: dqmgui/style/SiPixelRenderPlugin.cc [hunk 1]
```diff
@@ -269,7 +269,7 @@ class SiPixelRenderPlugin : public DQMRenderPlugin
 
       TH2F* obj2 = dynamic_cast<TH2F*>( o.object );
 
-      if( o.name.find("Summary") != std::string::npos && o.name.find("_Error_") == std::string::npos) 
+      if( o.name.find("Summary") != std::string::npos && o.name.find("_Error_") == std::string::npos && o.name.find("PixelPhase1/Summary")!=std::string::npos)
```
Comment (vanbesien) [inline dqmgui/style/SiPixelRenderPlugin.cc]: Hey, are you sure about this? You're basically removing the reportSummaryMapPalette for the original Summary with this line.

Comment (fioriNTU) [inline dqmgui/style/SiPixelRenderPlugin.cc]: right! I will fix now, I tend to forget about Phase 0, thanks for the notice

Comment (fioriNTU) [inline dqmgui/style/SiPixelRenderPlugin.cc]: Actually, it is not true, I have just checked, few lines below you can find the actual Phase 0 summary called "reportSummaryMap" that has the palette set correctly. I think that after my latest commit (that is not affecting online) this is ok.

**Comment by threus** [comment]
@BrunoCoimbra @h4d4 could you please merge this PR?
Thanks,
Tomas for DQM

**Pull request merged**